client/src/pages/Profile.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Container, Row, Col } from 'react-bootstrap';
+import SolutionCard from '../components/SolutionCard';
+import "./Profile.css"
+
+const Profile: React.FC = () => {
+  const [userData, setUserData] = useState<any>({});
+  const [recentSolutions, setRecentSolutions] = useState<any[]>([]);
+	const [recentComments, setRecentComments] = useState<any[]>([]);
+
+	// Hardcode recentSolutions and recentComments for now.
```

Comment:
Rewrite it using the API. Use `fetch`. Uninstall `axios`.

Reply:
We will worry about this later once we get Shibolleth figured out